rocky/rocky/views/mixins.py

```diff
@@ -314,18 +314,18 @@ def __getitem__(self, key: int | slice) -> list[HydratedReport | Report]:
         raise NotImplementedError("ReportList only supports slicing")
 
     def get_subreports(self, report_id: str) -> list[Report]:
```

Comment:
Hi, this whole method is not needed as the get_reports already returned tuples with child and parent. It causes huge huge performance drawbacks doing these queries in Rocky instead of in Octopoes (where they should be done). Please see the return types of list_reports in Octopoes where you'll find that all reports are always returned together with their children.